integration-tests/Tests/IntegrationTests/PrimitiveTests.swift

```diff
@@ -0,0 +1,28 @@
+import RustLib
```

Comment:
This is just a reformat of the existing test, plus this `import RustLib` was added. If you'd like me to take a stab at porting it to the new [Swift Testing](https://developer.apple.com/xcode/swift-testing/) framework, let me know.